website/src/css/custom.css

```diff
@@ -4,31 +4,31 @@
  * work well for content-centric websites.
  */
 
-/* You can override the default Infima variables here. */
+/* Override infima variables */
 :root {
-    --ifm-color-primary: #2e8555;
-    --ifm-color-primary-dark: #29784c;
-    --ifm-color-primary-darker: #277148;
-    --ifm-color-primary-darkest: #205d3b;
-    --ifm-color-primary-light: #33925d;
-    --ifm-color-primary-lighter: #359962;
-    --ifm-color-primary-lightest: #3cad6e;
+    --ifm-color-primary: #f04d21;
+    --ifm-color-primary-dark: #e63d10;
+    --ifm-color-primary-darker: #d93a0f;
+    --ifm-color-primary-darkest: #b3300c;
+    --ifm-color-primary-light: #f2623b;
+    --ifm-color-primary-lighter: #f36c47;
+    --ifm-color-primary-lightest: #f58b6e;
```

Comment:
Changed theme colors to some Formidable colors. If you want to change these, [this is what I used](https://docusaurus.io/docs/styling-layout#styling-your-site-with-infima) to generate the variables.